distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -162,6 +162,7 @@ ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controll
 [5.0.0]
 ALERT;Guntamatic Binding: The Binding now uses channel groups and the dynamic status channels now have an index prefix to allow multiple channels with the same description. Existing Items will need to be adjusted.
 ALERT;MQTT Binding (Home Assistant): Thing types and channel IDs for things created prior to 4.3.0 have been significantly restructured and simplified. Items will need to be re-linked. Delete and re-create your Things to also have a simplified Thing Type ID in your Thing IDs.
+ALERT;CORE: Persistence alias functionality has been improved, allowing defining aliases across strategies and filter criteria for a persistence service. Previous persistence configurations using aliases need to be reconfigured.
```

Comment:
Maybe it would be helpful to directly add a link to the according documentation page?

Reply:
I don't think the adjusted documentation is available online yet. And the link will change with the release.